modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -257,6 +260,14 @@ When forwarding logs to external services, additional metadata might be availabl
 | labels_type | Specifies the type of deployment that the pod is running. | MuleApplication
 |===
 
+== Send a Test Message
+
+You can send a test message to verify connectivity between Runtime Fabric and the third-party log forwarding solution.
```

Comment:
third-party -> external

Reply:
As long as we are consistent, we can use either term. Do you want this changed throughout?